floodviz/tests/hydrograph_utils_tests.py

```diff
@@ -87,27 +87,26 @@ def setUp(self):
                 "key": "05463500",
                 "name": "Black Hawk Creek at Hudson, IA",
                 "date": "2008-05-20",
-                "time": "00:15:00",
-                "time_mili": 1211260500000.0,
-                "value": "446"
-            },
-            {
-                "key": "05463500",
-                "name": "Black Hawk Creek at Hudson, IA",
-                "date": "2008-05-20",
                 "time": "00:30:00",
                 "time_mili": 1211261400000.0,
                 "value": "446"
             }]
 
 
-    def test_empty_data(self):
-        self.assertEqual(parse_hydrodata([]), [])
-        self.assertEqual(parse_hydrodata(None), [])
+def test_empty_data(self):
+    self.assertEqual(parse_hydrodata([]), [])
+    self.assertEqual(parse_hydrodata(None), [])
 
-    def test_data_not_list(self):
-        self.assertEqual(parse_hydrodata({}), [])
+def test_data_not_list(self):
+    self.assertEqual(parse_hydrodata({}), [])
 
+def test_valid_data(self):
```

Comment:
This should be removed because the function is properly declared on the next line.
Also, this test fails even after that change.